libp2p/transports/quictransport.nim

```diff
@@ -155,12 +171,27 @@ method start*(
 ) {.async: (raises: [LPError, transport.TransportError]).} =
   doAssert self.listener.isNil, "start() already called"
   #TODO handle multiple addr
+
+  let pubkey = self.privateKey.getPublicKey().valueOr:
+    doAssert false, "could not obtain public key"
+    return
+
+  let keypair = KeyPair(seckey: self.privateKey, pubkey: pubkey)
+
+  let certPair = generate(keypair, EncodingFormat.PEM)
```

Comment:
- `generate` what? it's deductible from context that certificate is created but better naming would be appreciated. 
- also `generate` returns tuple, but why not named tuple? or why not simple type? usage like `certPair[0]`, `certPair[1]` is not really readable and understandable unless you go into the function.
 
 
i know it's not related to this pr, but if you agree we can create issue for this, ideal for  "good first issues" lable.